### ISSUE TITLE
Frame drops

### ISSUE THREAD

**Issue by LinleyLin** [Resike/Z-Perl#190]
For multible months now I encounter massive frame drops in raids when I use Z-Perl. If I disable Z-Perl, I am running on up to 120 frames in raids (I set max frames to 120). With Z-Perl on, it drops down to 22 at each boss, no matter which raid or boss. 
I tried to narrow it down by disabling Z-Perl and then enabling it one by one, about half of the drop happens through raid and party frames. I really have no idea why this is happening, especially since the game itself runs smoothly with the addon disabled.

**Comment by ThisIsPita** [comment]
Same, massive frame drops when ZPerl is on. Used bugsack addon but did not find any LUA errors. Really unfortunate as I love the addon and have been using it since X-Perl stopped being maintained.

**Comment by LinleyLin** [comment]
Yes, I absolutely love ZPerl! Been using it ever since and I really don't want to stop.
There is no other addon which looks this good or has that many useful functions, so I hope there is a solution for this issue  :(
I can provide details on my PC-setup if needed. 
The more incoming information Z-Perl has to work with (raidgroups), the worse the frame drop gets for some reason

**Comment by zeenk** [comment]
Go in Z-Perl settings and to the All tab and disable 3d portraits, it helps a lot with fps.
I think there might be an issue with model textures atm, weakauras gave me a warning about that

![2024-11-02_21-34-43](https://github.com/user-attachments/assets/f2a9adcf-7f51-467f-9e22-56e4ce4fe40a)

**Comment by ThisIsPita** [comment]
> Go in Z-Perl settings and to the All tab and disable 3d portraits, it helps a lot with fps. I think there might be an issue with model textures atm, weakauras gave me a warning about that
> 
> ![2024-11-02_21-34-43](https://private-user-images.githubusercontent.com/68170439/382507030-f2a9adcf-7f51-467f-9e22-56e4ce4fe40a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA1ODQ4MjcsIm5iZiI6MTczMDU4NDUyNywicGF0aCI6Ii82ODE3MDQzOS8zODI1MDcwMzAtZjJhOWFkY2YtN2Y1MS00NjdmLTllMjItNTZlNGNlNGZlNDBhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTAyVDIxNTUyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYwYmI0OWE5ZWRhMDcwMmQ5OGMxNmY4NGQ2YjcyNjQ4OTM0ZWMyMTBjODdiODhkNDAxOTU4M2VlNDg1MGYyOTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.2JahxyGqCX19NhOcyNi4jWw6W6i2E3DYQTaupLr2bx0)

Thanks! I'll try this.

**Comment by LinleyLin** [comment]
Thanks for the hint, but It odes not solve the issue. Like I said, its not a recently occured thing, it has been like this for months, longer than the newest expansion is out
